src/spikeinterface/core/job_tools.py

```diff
@@ -359,7 +359,27 @@ def __init__(
         self.max_threads_per_process = max_threads_per_process
 
         if verbose:
-            print(self.job_name, "with n_jobs =", self.n_jobs, "and chunk_size =", self.chunk_size)
+            if self.n_jobs > 1:
+                chunk_memory = self.chunk_size * recording.get_num_channels() * np.dtype(recording.get_dtype()).itemsize
+                total_memory = chunk_memory * self.n_jobs
+                chunk_duration = self.chunk_size / recording.get_sampling_frequency()
+                from spikeinterface.core.core_tools import convert_bytes_to_str, convert_seconds_to_str
+
+                chunk_memory_str = convert_bytes_to_str(chunk_memory)
+                total_memory_str = convert_bytes_to_str(total_memory)
+                chunk_duration_str = convert_seconds_to_str(chunk_duration)
+                print(
+                    self.job_name,
+                    "\n"
```

Comment:
not sure 2 lines is a good idea because with the progressbar this will be 3 lines in total.

Reply:
I am fine with reducing it to two.

Reply:
Reverted to one line.

![image](https://github.com/SpikeInterface/spikeinterface/assets/6429509/e7d96a95-7924-4c55-b779-087980084ee0)